packages/ses/scripts/bundle.js

```diff
@@ -1,10 +1,7 @@
 /* eslint-disable import/first */
 /* global process */
-
-import '../index.js';
-
 lockdown();
-
+import '../index.js';
```

Comment:
IIUC, this does not actually change the order in which things happen, at least in the absence of cyclic imports. All imported modules are evaluated/initialized before the code in this module starts evaluation/initialization. So either way, `../index.js` initialization will happen before this call to `lockdown`.

Attn @kriskowal

Reply:
This is true. To interleave behavior relative to other imports, we have to use a module like `lockdown.js` that just calls `lockdown()`. So,

```js
import '../index.js'; // or maybe import 'ses' reflexively works now
import './lockdown.js'; // just lockdown()
```

This insures that `./index.js` initializes before you call `lockdown()`.

It’s unclear why the tests pass with this defect. It might be that we need to test `yarn build` under `yarn test`.

Reply:
makes sense ^ reverted [feat(ses): call lockdown earlier](https://github.com/endojs/endo/pull/2337/commits/b5c04714ebdfa5a36598b0b21f9c3d31d0525c73), looks good now

Reply:
> It’s unclear why the tests pass with this defect. It might be that we need to test `yarn build` under `yarn test`.

yeah the tests still pass calling lockdown() too early before the imports 🤔

https://github.com/endojs/endo/blob/7a2955080f34880c8028e1b2dff06bf7b969790f/packages/ses/package.json#L75

even when doing a clean build before `"test": "yarn prepare && tsd && ava",` if that's what we meant